### PR TITLE
Leave blank line before closing comment if idented

### PR DESCRIPTION
**Pull request by eksperimental**
If last line in the docs is idented, leave an emptyline before closing the comment.

Mentioned by @whatyouhide in https://github.com/elixir-lang/elixir/pull/7606#discussion_r184348514

The command/regexp used to find these offending lines is:

    ag "\ {4,}[^\n]+\n\s{2}[\"']{3}"

### CODE REVIEW

File: lib/mix/lib/mix/tasks/new.ex [hunk 1]
```diff
@@ -408,6 +408,7 @@ defmodule Mix.Tasks.New do
   # here (which is why it is important to import them last).
   #
   #     import_config "#{Mix.env}.exs"
+  #
```
Comment (whatyouhide) [inline lib/mix/lib/mix/tasks/new.ex]: We shouldn't have this line in this template.

File: lib/mix/lib/mix/tasks/new.ex [hunk 2]
```diff
@@ -428,6 +429,7 @@ defmodule Mix.Tasks.New do
   #       level: :info,
   #       format: "$date $time [$level] $metadata$message\n",
   #       metadata: [:user_id]
+  #
```
Comment (whatyouhide) [inline lib/mix/lib/mix/tasks/new.ex]: We shouldn't have this line in this template.

File: lib/mix/lib/mix/tasks/new.ex (hunk 1 above)
Comment (eksperimental) [reply lib/mix/lib/mix/tasks/new.ex]: true

File: lib/mix/lib/mix/tasks/new.ex (hunk 2 above)
Comment (eksperimental) [reply lib/mix/lib/mix/tasks/new.ex]: oops

**Comment by eksperimental** [comment]
@whatyouhide fixed

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**